django_loci/apps.py

```diff
@@ -15,11 +25,21 @@ def __setmodels__(self):
         from .models import Location
         self.location_model = Location
 
+    def test_geocoder(self):
+        try:
+            location = geocode('Red Square')
+            assert location is not None
+        except AssertionError:
+            if DJANGO_LOCI_GEOCODE_STRICT_TEST:
+                raise ImproperlyConfigured('Geocode utility is improperly configured')
+            self.logger.exception('Geocode test failed')
```

Comment:
same here, I think this should output the same text as the line before, in the first case the app will be stopped and the exception raised, in this other case the exception will be logged but the application startup will not be halted